draft-packages/modal/KaizenDraft/Modal/Primitives/ModalFooter.tsx

```diff
@@ -35,6 +36,7 @@ const ModalFooter: ModalFooter = props => {
               destructive={index === 0 && appearance === "destructive"}
               secondary={index > 0}
               disabled={a.disabled}
+              automationId={`${automationId}-action-${index}`}
```

Comment:
Question, and not a blocker - but I'm curious about the name. I guess since you're setting the data attributes on the wrapping div around this so you need a different name, but I'm not sure if we have a precedent/pattern for the convention yet. Why `action`?

Reply:
I used `action` just because that's what they're called in the ModalFooter props: `actions`. Would you prefer `button`?